2.4/root/usr/bin/run-mongod

```diff
@@ -74,26 +74,25 @@ if [ -z "${MONGODB_REPLICA_NAME-}" ]; then
     usage
   fi
   # Run the MongoDB in 'standalone' mode
-  if [ ! -f ${MONGODB_DATADIR}/.mongodb_datadir_initialized ]; then
-    # Create MongoDB users and restart MongoDB with authentication enabled
-    # At this time the MongoDB does not accept the incoming connections.
-    mongod $mongo_common_args & #--bind_ip 127.0.0.1 --quiet >/dev/null &
-    wait_for_mongo_up
+  mongod $mongo_common_args &
+  wait_for_mongo_up
+  js_command="db.system.users.count({'user':'admin'})"
+  if [ "$(mongo admin --quiet --eval "$js_command")" == "1" ]; then
```

Comment:
Yeah, not to check in every command, just thinking aloud to make sure we're not accidentally changing behavior.

Checking if a file exists fails for different reasons than `mongo admin --quiet --eval ...`.
Perhaps I'm just thinking this far remembering debug sessions that @php-coder had when we saw strange behavior hard to correlate with an actual cause.

We have `set -e` etc to make it fail fast, I wouldn't like to add `|| true` every other line, because IMHO it makes it harder to understand problems when things blow up much further from the original error.